scene/3d/mesh_instance.cpp

```diff
@@ -414,7 +414,10 @@ void MeshInstance::_update_skinning() {
 		const int index_count_write = software_skinning_mesh->surface_get_array_index_len(surface_index);
 
 		uint32_t array_offsets_write[Mesh::ARRAY_MAX];
-		const uint32_t stride_write = visual_server->mesh_surface_make_offsets_from_format(format_write, vertex_count_write, index_count_write, array_offsets_write);
+		uint32_t array_strides_write[Mesh::ARRAY_MAX];
+		visual_server->mesh_surface_make_offsets_from_format(format_write, vertex_count_write, index_count_write, array_offsets_write, array_strides_write);
+		const uint32_t position_stride_write = array_strides_write[Mesh::ARRAY_VERTEX];
+		const uint32_t attribute_stride_write = array_strides_write[Mesh::ARRAY_NORMAL];
```

Comment:
Maybe just have something like:
```
const uint32_t stride_write = array_strides_write[Mesh::ARRAY_VERTEX];
ERR_FAIL(stride_write != array_strides_write[Mesh::ARRAY_NORMAL]);
```
That enables us to quickly find bugs where the wrong split stream is being used with dynamic use, and ensures the hot loop works as intended.